.gitignore

```diff
@@ -1,6 +1,6 @@
-build/???/
-build/premake/**/bin/
-build/premake/**/obj/
+/*build*/
```

Comment:
Why the wildcards? Wouldn't be a simple /build/ enough?

Reply:
it matches any of `/_build` `/builddebug` etc